### ISSUE TITLE
Settings App Redesign

### ISSUE THREAD

**Issue by anuragk47** [materialos/CM12-Theme#40]
To do:
- [x] New card like layout inspired from Google Now
- [x] New color scheme
- [x] Fix no card layout on CM13
- [x] Fix bg color in subsettings on CM13
- [x] Fix switchbar padding issues
- [x] Fix some icons colored differently on CM 13
- [x] Add all icons in vector format, this includes adding ROM specific icons.
- [x] Make sure ROM specific sub settings are themed (like gestures)
- [x] Update companion app themes like cyanogenmod updater, etc
- [x] Finer details
- [x] Finally, test on popular ROMs for bugs and only after fixing them close this issue.

**Comment by anuragk47** [comment]
This is still buggy on CM 13. I'll have to look into it and also add all icons in vector format.

**Comment by anuragk47** [comment]
Commit f893888 fixed card layout on CM13 but caused darker bg in subsettings

**Comment by anuragk47** [comment]
The icon colors on CM13 can only be fixed by adding each one of the settings icons, since they are defined to use attr/colorAccent by default. Will also have to add icons for popular custom roms

**Comment by anuragk47** [comment]
Done for now. Need to keep adding some custom ROM stuff.
